### DISCUSSION TITLE
2022.6.0

### DISCUSSION THREAD

**Discussion by Thomas55555** [Thomas55555/husqvarna_automower#255] [Announcements]
![downloads](https://img.shields.io/github/downloads/Thomas55555/husqvarna_automower/2022.6.0/total.svg)
There are a lot of new statistics sensors now, like you have them in the AutomowerApp and also some additional (number of collisions, cutting blade usage time) which are disabled by default.
I've your mower support information about the last change of the cutting blades, please let me know. So I can enable this feature only for mowers, where this is working (e.g. works not for my 315X).

## What’s Changed

* tweak SensorEntityDescription (#254) @Thomas55555
* add percentage sensors (#253) @Thomas55555
* use timedelta for timestatistics (#252) @Thomas55555
* add state_class to statistics (#251) @Thomas55555
* add redirect information (#250) @Thomas55555
* bump aioautomower to 2022.6.0 (#249) @Thomas55555
* Add statistic sensors (#248) @Thomas55555
* add Estonian translation (#242) @priiduonu
* remove domain from hacs.json (#241) @Thomas55555
* add mower name to calendar (#240) @Thomas55555

<hr /><em>This discussion was created from the release <a href='https://github.com/Thomas55555/husqvarna_automower/releases/tag/2022.6.0'>2022.6.0</a>.</em>

**Comment by tomcatxx** [comment]
Hi the 415x supports the following disables sensors:
sensor.am415x_cutting_blade_usage_time
sensor.415x_number_of_collisions
sensor.am415x_total_cutting_time
sensor.am415x_total_searching_time
binary_sensor.am415x_battery_charging
binary_sensor.am415x_leaving_dock
sensor.am415x_mode
tnx for the good work!

**Reply by Thomas55555** [reply]
In what unit in the cutting blade usage time given? In seconds?
And does anyone knows, what number of collisions mean?

**Reply by tomcatxx** [reply]
Here I made scom screenshots with all informations :)
https://ibb.co/hm82KXz
https://ibb.co/mF9ssW1

**Comment by priiduonu** [comment]
number of collisions is probably the number of "bumps" against any obstacles? mine is around 35000 since last june.
